apps/aeoracle/src/aeo_query.erl

```diff
@@ -173,6 +180,26 @@ serialization_template(?ORACLE_QUERY_VSN) ->
     , {fee            , int}
     ].
 
+-spec serialize_for_client(query()) -> map().
+serialize_for_client(#query{} = I) ->
+    {delta, ResponseTtlValue} = response_ttl(I),
+    Response = case response(I) of
+                   undefined -> <<>>;
+                   R -> R
+               end,
+    #{ <<"query_id">>     => aec_base58c:encode(oracle_query_id, id(I))
+     , <<"sender">>       => aec_base58c:encode(account_pubkey, sender_address(I))
+     , <<"sender_nonce">> => sender_nonce(I)
+     , <<"oracle_id">>    => aec_base58c:encode(oracle_pubkey, oracle_address(I))
+     , <<"query">>        => query(I)
+     , <<"response">>     => Response
```

Comment:
This can be any arbitrary binary. It feels like it needs an encoding.

Reply:
cc @jur0

Reply:
✅